ui/intermittent-failures/redux/reducers.js

```diff
@@ -13,6 +13,7 @@ export const fetchData = (name = '') => (state = { data: {}, message: '' }, acti
       return {
         ...state,
         message: action.message,
+        status: action.status,
```

Comment:
I meant to comment here earlier, but I think it'd be clearer to change `status` to `failureStatus` since its only being assigned a value if there's a failure.